R2API/Utils/Reflection.cs

```diff
@@ -923,5 +925,60 @@ public static class Reflection {
         }
 
         #endregion
+
+        public static System.Reflection.FieldInfo GetNestedField(Type type, string fieldName) {
+            var nestedTypes = type.GetNestedTypes(AllFlags);
+            foreach (Type nestedType in nestedTypes) {
+                var fieldInfo = nestedType.GetField(fieldName, AllFlags);
+                if (fieldInfo != null) {
+                    return fieldInfo;
+                }
+            }
+            return null;
+        }
+
+        public static System.Reflection.MethodInfo GetNestedMethod(Type type, string methodName) {
+            var nestedTypes = type.GetNestedTypes(AllFlags);
+            foreach (Type nestedType in nestedTypes) {
+                var methodInfo = nestedType.GetMethod(methodName, AllFlags);
+                if (methodInfo != null) {
+                    return methodInfo;
+                }
+            }
+            return null;
+        }
+
+        public static void LogCursorOpcodes(ILCursor cursor) {
```

Comment:
What's the difference between this and just logging the cursor to string?

Reply:
I didn't know you could just log the cursor to a string. If you can do that then I agree that this function is redundant.